src/ripple/test/jtx/impl/Env_test.cpp

```diff
@@ -561,6 +561,21 @@ class Env_test : public beast::unit_test::suite
                 );
     }
 
+    void testSig()
+    {
+        using namespace jtx;
+        Env env(*this);
+
+        env.fund(XRP(10000), "alice");
+        auto const baseFee = env.app().config().FEE_DEFAULT;
+        std::uint32_t const aliceSeq = env.seq ("alice");
+        
+        Json::Value jsonNoop = env.json (
+            noop ("alice"), fee(baseFee), seq(aliceSeq), sig("alice"));
+        JTx jt = env.jt (jsonNoop);
+        env (jt);
```

Comment:
Short answer: Comments are probably sufficient.

Longer answer: Yes, without the change in utility.cpp, all 5 of my test cases should fail, because they all start with a signed transaction. The reason I wrote them up was because i was exploring the mechanism of _why_ your test case passes with the change, and noticed that your test transaction's signature was being ignored in favor of re-signing. My test cases demonstrate that a bit more (particularly the second, which is signed by "bob"), but those probably *do* cross over into business logic, and thus may be more relevant elsewhere.